src/lib/Zikula/Bundle/CoreInstallerBundle/Command/UpgradeResetCommand.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/*
+ * This file is part of the Zikula package.
+ *
+ * Copyright Zikula Foundation - http://zikula.org/
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Zikula\Bundle\CoreInstallerBundle\Command;
+
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Zikula\Bundle\CoreBundle\YamlDumper;
+
+class UpgradeResetCommand extends ContainerAwareCommand
```

Comment:
I think it would be more appropriate not to add this command but update the version using a shell command inside the travis.yml file.